src/hooks/govwallet/useGovWalletBalance/useGovWalletBalance.tsx

```diff
@@ -73,8 +73,12 @@ export const useGovWalletBalance = (
 
           // We only check the eligibility of the balance of the first account
           const areAllBalancesEligible = results.every(
-            // Check if balance is strictly equals to 10000 cents
-            ({ accountDetails }) => accountDetails[0].balance === 10000
+            // if govwalletExactBalanceValue == -1 or undefined, dont check exact value
```

Comment:
its abit awkward because for non exact match checks, i cant check if the balance is sufficient because the amount to deduct is stored in app config and mobile doesnt currently have access to it. however, since non exact match checks doesnt currently have a use case and its more for future proofing, i dont think theres a need to spend the extra effort to add app config to mobile in this story